File: fmarket/src/main/java/ro/fmarket/admin/demand/DemandSearchObject.java
package ro.fmarket.admin.demand;

import lombok.Data;
import ro.fmarket.model.demand.consts.DemandStatus;

@Data
public class DemandSearchObject {

	private Integer page;
	
	private Integer accountId;
	
	private Integer domainId;
	
	private DemandStatus status;
	
	
}

File: fmarket/src/main/webapp/app/models/DemandSearchObject.ts
/**
 * Created by NicolaeB on 5/26/2016.
 */
export class DemandSearchObject {
    accountId: string;
    page: number;
    status: string;
    domainId: number;
    domainName: string;

    constructor(accountId:string, page:number, status:string, domainId:number) {
        this.accountId = accountId;
        this.page = page;
        this.status = status;
        this.domainId = domainId;
    }
}